controllers/users.js

```diff
@@ -565,20 +576,45 @@ const markUnverified = async (req, res) => {
  * @param res {Object} - Express response object
  */
 
-const getUserImageForVerification = async (req, res) => {
+const getUserPhotoVerificationRequests = async (req, res) => {
   try {
-    const { id: userId } = req.params;
-    const userImageVerificationData = await userQuery.getUserImageForVerification(userId);
+    const { userId } = req.params;
+    const userData = req.userData;
+    if ((userData.id !== userId && !userData.roles[ROLES.SUPERUSER]) || !userId) {
+      return res.boom.unauthorized("You are not authorized to view this user's image verification data");
+    }
+    const userImageVerificationData = await userQuery.getUserPhotoVerificationRequests(userId);
     return res.json({
       message: "User image verification record fetched successfully!",
-      data: userImageVerificationData,
+      data: userImageVerificationData[0],
     });
   } catch (error) {
-    logger.error(`Error while verifying image of user: ${error}`);
+    logger.error(`Error while querying image of user: ${error}`);
```

Comment:
Super users is allowed, and users self is allowed right?

Reply:
Yes, in the case of the SUPERUSER they can query the data of any user, but for any other user-role they can only query their own data